server/models/PersonalToken.ts

```diff
@@ -0,0 +1,92 @@
+import { randomBytes } from 'crypto';
+
+import { CreationOptional, InferAttributes, InferCreationAttributes, NonAttribute } from 'sequelize';
+
+import oAuthScopes from '../constants/oauth-scopes';
+import sequelize, { DataTypes, Model } from '../lib/sequelize';
+
+import models from '.';
+
+class PersonalToken extends Model<InferAttributes<PersonalToken>, InferCreationAttributes<PersonalToken>> {
+  public declare readonly id: CreationOptional<number>;
+  public declare token: string;
+  public declare expiresAt: Date;
+  public declare createdAt: CreationOptional<Date>;
+  public declare updatedAt: CreationOptional<Date>;
+  public declare deletedAt: CreationOptional<Date>;
+  public declare data: Record<string, unknown>;
+  public declare CollectiveId: number;
+  public declare UserId: number;
```

Comment:
:ice_cream: No need to iterate in this PR, but since `User` is now converted to TS we can do:

```suggestion
  public declare UserId: ForeignKey<User['id']>;
```